pom.xml

```diff
@@ -550,6 +550,7 @@
         <version.org.jboss.metadata>16.0.0.Final</version.org.jboss.metadata>
         <version.org.jboss.mod_cluster>2.0.4.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>7.1.0.Final</version.org.jboss.narayana>
```

Comment:
Please notice that narayana 7.1.0.Final is still present because Narayana LRA is now independently released